analytic_partner_hr_timesheet/models/account_analytic_line.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class AnalyticAccountLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    @api.onchange('project_id')
+    def on_change_project_id(self):
+        if self.project_id:
```

Comment:
Yes, that's true, the method has not make sense since "Partner" is the partner of the analytic account. Some options (at least):
- Delete the method
- Include the analytic account in the view (as it was in V8) and keep the method as it was (on change over the analytic account field)
What do you think?